docs/sources/administration/api-keys/index.md

```diff
@@ -27,11 +27,24 @@ Create an API key when you want to manage your computed workload with a user.
 
 This topic shows you how to create an API key using the Grafana UI. You can also create an API key using the Grafana HTTP API. For more information about creating API keys via the API, refer to [Create API key via API]({{< relref "../../developers/http_api/create-api-tokens-for-org/#how-to-create-a-new-organization-and-an-api-token" >}}).
 
-### Before you begin:
+### Before you begin
+
+To follow these instructions, you need:
+
+Either
+
+- Admin/Editor
+
+Or
+
+- API keys writer
+- Service account writer
```

Comment:
@chri2547 you have a better way of presenting this? 🤔

Reply:
Is it API keys writer AND service account writer? Or just one of the two?

Reply:
it's actually just service account writer/creator